### ISSUE TITLE
Could not pass event UseBlockEvent to ShopChest

### ISSUE THREAD

**Issue by FataLeeer** [EpicEricEE/ShopChest#57]
```
[Server thread/ERROR]: Could not pass event UseBlockEvent to ShopChest v1.11.1
org.bukkit.event.EventException
	at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:306) ~[spigot.jar:git-Spigot-f950f8e-aa25753]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[spigot.jar:git-Spigot-f950f8e-aa25753]
	at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:502) [spigot.jar:git-Spigot-f950f8e-aa25753]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:487) [spigot.jar:git-Spigot-f950f8e-aa25753]
	at com.sk89q.worldguard.bukkit.util.Events.fireToCancel(Events.java:74) [worldguard-legacy-6.2.1-SNAPSHOT-dist.jar:?]
	at com.sk89q.worldguard.bukkit.listener.EventAbstractionListener.onInventoryOpen(EventAbstractionListener.java:783) [worldguard-legacy-6.2.1-SNAPSHOT-dist.jar:?]
	at sun.reflect.GeneratedMethodAccessor279.invoke(Unknown Source) ~[?:?]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_111]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_111]
	at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:302) [spigot.jar:git-Spigot-f950f8e-aa25753]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) [spigot.jar:git-Spigot-f950f8e-aa25753]
	at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:502) [spigot.jar:git-Spigot-f950f8e-aa25753]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:487) [spigot.jar:git-Spigot-f950f8e-aa25753]
	at org.bukkit.craftbukkit.v1_11_R1.event.CraftEventFactory.callInventoryOpenEvent(CraftEventFactory.java:753) [spigot.jar:git-Spigot-f950f8e-aa25753]
	at net.minecraft.server.v1_11_R1.EntityPlayer.openContainer(EntityPlayer.java:689) [spigot.jar:git-Spigot-f950f8e-aa25753]
	at net.minecraft.server.v1_11_R1.BlockShulkerBox.interact(BlockShulkerBox.java:57) [spigot.jar:git-Spigot-f950f8e-aa25753]
	at net.minecraft.server.v1_11_R1.PlayerInteractManager.a(PlayerInteractManager.java:483) [spigot.jar:git-Spigot-f950f8e-aa25753]
	at net.minecraft.server.v1_11_R1.PlayerConnection.a(PlayerConnection.java:897) [spigot.jar:git-Spigot-f950f8e-aa25753]
	at net.minecraft.server.v1_11_R1.PacketPlayInUseItem.a(PacketPlayInUseItem.java:37) [spigot.jar:git-Spigot-f950f8e-aa25753]
	at net.minecraft.server.v1_11_R1.PacketPlayInUseItem.a(PacketPlayInUseItem.java:1) [spigot.jar:git-Spigot-f950f8e-aa25753]
	at net.minecraft.server.v1_11_R1.PlayerConnectionUtils$1.run(SourceFile:13) [spigot.jar:git-Spigot-f950f8e-aa25753]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_111]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_111]
	at net.minecraft.server.v1_11_R1.SystemUtils.a(SourceFile:46) [spigot.jar:git-Spigot-f950f8e-aa25753]
	at net.minecraft.server.v1_11_R1.MinecraftServer.D(MinecraftServer.java:739) [spigot.jar:git-Spigot-f950f8e-aa25753]
	at net.minecraft.server.v1_11_R1.DedicatedServer.D(DedicatedServer.java:399) [spigot.jar:git-Spigot-f950f8e-aa25753]
	at net.minecraft.server.v1_11_R1.MinecraftServer.C(MinecraftServer.java:675) [spigot.jar:git-Spigot-f950f8e-aa25753]
	at net.minecraft.server.v1_11_R1.MinecraftServer.run(MinecraftServer.java:574) [spigot.jar:git-Spigot-f950f8e-aa25753]
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_111]
```

**Comment by EpicEricEE** [comment]
Fixed with 4f894dd

**Comment by FataLeeer** [comment]
how to download 4f894dd?
